tests/.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MASTER]
+# Set the Python version
+py-version=3.13
+
+[MESSAGES CONTROL]
+disable=
```

Comment:
A couple of these seem like things we'd want to lint for. The two that jump out at me are checking for docstrings and duplicate code